src/bootstrap.cc

```diff
@@ -2,7 +2,7 @@
 
 #include <boost/algorithm/string.hpp>
 
-#include <stdio.h>
+#include <cstdio>
```

Comment:
Isn't this asking for trouble since we aren't prefixing the C functions with `std::`? (See https://stackoverflow.com/questions/10460250/cstdio-stdio-h-namespace and the `fmemopen` below.)

Reply:
yes You are right - https://clang.llvm.org/extra/clang-tidy/checks/modernize-deprecated-headers.html

Reply:
We need to prepend all c functions with std::

Reply:
I replaced all changes back